### PR TITLE
update qgis_gui.rst

### PR DESCRIPTION
**Pull request by dragons8mycat**
fix typos and grammar - update to US-EN

### CODE REVIEW

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 1]
```diff
@@ -367,17 +367,9 @@ Map Legend
...
-* |mActionFilterMap| **Filter Legend by Map Content**: only the layers that are set visible and whose
-  features intersects the current map canvas have their style rendered in the map
-  legend. Otherwise, a generic NULL symbol is applied to the layer. This is a
-  convenient way to identify which layers cover your area of interest.
-* |mIconExpressionFilter| **Filter Legend by Expression**
-* |mActionExpandTree| **Expand All** or |mActionCollapseTree| **Collapse All** layers and groups in the map legend
-* and |mActionRemoveLayer| **Remove** the selected **Layer or Group**.
+map legend are list allow you to **Add group**, **Manage Layer Visibility** of 
+all layers or manage preset layers combination, **Filter Legend by Map 
+Content**, **Expand All** or **Collapse All** and **Remove Layer or Group**.
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/qgis_gui.rst]: Just by curiosity, why did you remove list of actions? Is it too much details?

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 2]
```diff
@@ -626,13 +609,11 @@ loading, processing tools...)
...
-   When you start QGIS, the default CRS is ``WGS 84 (epsg 4326)`` and 
-   units are degrees. This means that QGIS will interpret any
-   coordinate in your layer as specified in degrees. To get correct scale values,
-   you can either manually change this setting, e.g. to meters, in the :guilabel:`General`
-   tab under :menuselection:`Project --> Project Properties`, or you can use
-   the |mIconProjectionDisabled| :sup:`Current CRS:` icon seen above. In the latter case,
-   the units are set to what the project projection specifies (e.g., ``+units=us-ft``).
-   
-   Note that CRS choice on startup can be set in :menuselection:`Settings --> Options --> CRS`.
+   When you start |qg|, the default units are degrees, this means that |qg| will interpret any
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/qgis_gui.rst]: you can replace |qg| by QGIS everywhere.

**Comment by yjacolin** [comment]
waiting for the test. Looks good to me. You can comment my comments if you think it is worth to answer :)

**Comment by dragons8mycat** [comment]
Please don't commit, not all those changes are mine. I didn't remove the list for starters

**Comment by dragons8mycat** [comment]
Apologies, took time to look over a few of those changes....I'm confused. Will remove everything and start from scratch. This is getting really frustrating....but the changes to the CRS references I definitely left in.
